ci-operator/jobs/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master-postsubmits.yaml

```diff
@@ -1,20 +1,16 @@
-presubmits:
+postsubmits:
   openshift-cnv/cnv-ci:
   - agent: kubernetes
-    always_run: true
     branches:
-    - master
+    - ^master$
```

Comment:
maybe we should use other branch than master to push there the cvp bundle updates.
a branch per CNV minor release.

Reply:
I've created two branches in the source repo and two jobs to separated the updates per version.